apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,407 @@
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// from columns whose families satisfy the given RE2 regex.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// For technical reasons, the regex must not contain the `:` character, even
+        /// if it is not being used as a literal. Note that, since column families
+        /// cannot contain the new line character `\n`, it is sufficient to use `.`
+        /// as a full wildcard when matching column family names.
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match column families.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter FamilyNameRegex(string regex)
+        {
+            throw new InvalidOperationException();
+        }
+
+        // TODO: Copy docs from Interleave.Filters, but only once the formatting issues are resolved.
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which applies several
+        /// RowFilters to the data in parallel and combines the results.
+        /// </summary>
+        /// <param name="filters">
+        /// The filters to apply in parallel.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter Interleave(params RowFilter[] filters)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches all cells,
+        /// regardless of input. Functionally equivalent to leaving `filter` unset,
+        /// but included for completeness.
+        /// </summary>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter PassAllFilter()
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// from rows whose keys satisfy the given RE2 regex. In other words, passes
+        /// through the entire row when the key matches, and otherwise produces an
+        /// empty row.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that, since row keys can contain arbitrary bytes, the `\C` escape
+        /// sequence must be used if a true wildcard is desired. The `.` character
+        /// will not match the new line character `\n`, which may be present in a
+        /// binary key.
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match row keys.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter RowKeyRegex(RowKey regex)
```

Comment:
Again, string overload?

Reply:
As with `ColumnQualifier`, string can be used here.